client/components/Home/Home.jsx

```diff
@@ -1,180 +1,170 @@
-                        testers to record results.
-                    </p>
-                    <div className="hero-buttons">
-                        {isSignedIn ? (
-                            <></>
-                        ) : (
-                            <Button
-                                variant="primary"
-                                onClick={() =>
-                                    (window.location.href = signinUrl)
-                                }
+            <section className="hero-section">
+                <h1>
+                    Enabling Interoperability for Assistive Technology Users
+                </h1>
+                <div className="hero-copy-and-video">
+                    <div className="hero-copy">
+                        <p>
+                            Today, different screen readers often yield
+                            conflicting experiences when presenting a web page,
+                            disadvantaging or even excluding some users. These
+                            differences also create accessibility design and
+                            test barriers for web developers.
+                        </p>
+                        <p>
+                            On the other hand, browsers are interoperable for
+                            people who do not use assistive technologies. That
+                            is, different browsers provide equivalent
+                            experiences. Browser interoperability facilitates an
+                            inclusive web.
+                        </p>
+                        <p>
+                            Assistive technology users deserve equal inclusion.
+                            The ARIA-AT project aims to empower equal inclusion
+                            by realizing interoperability for AT users.
+                        </p>
+                        <p>
+                            <a
+                                className="hero-link"
+                                href="https://github.com/w3c/aria-at/wiki/How-Gaps-in-Assistive-Technology-Interoperability-Hinder-Inclusion"
                             >
-                                Sign Up to Run Tests
-                            </Button>
-                        )}
-                        <Link className="btn-secondary btn" to="/reports">
-                            Browse Test Reports
-                        </Link>
+                                Read more about how the AT interoperability gap
+                                hinders inclusion on the web for people with
+                                disabilities.
+                            </a>
+                        </p>
+                    </div>
+                    <div className="hero-video">
+                        <iframe
+                            src="https://player.vimeo.com/video/651279608?h=45aefd646f"
```

Comment:
Should be `https://player.vimeo.com/video/651279608?h=45aefd646f&byline=false&dnt=true&portrait=false` to hide byline and portrait (please verify on your end, I haven't been able to preview these query params

Reply:
Verified. Video looks awesome can't wait to watch the whole thing later 🤓